### ISSUE TITLE
CASIA-Webface, training acc is 0.99, but verification acc is 0.60+?

### ISSUE THREAD

**Issue by TccccD** [deepinsight/insightface#495]
CASIA-Webface, training acc is 0.99, but verification acc is 0.60+.
我看之前有类似的issue，#478, 但我用下面代码切分train和val，这应该可以让验证集分布和训练集一样
```
tmp = random.sample(range(a, b), (b-a)//10)
train_idx += [x for x in range(a, b) if x not in tmp]
val_idx += tmp
```
模型resnet50，per-batch-size=128，loss-type=4, 其他参数都没有变过；
需要调整下margin-m吗？0.35貌似也可以让training收敛，这是过拟合？
@nttstar

**Comment by TccccD** [comment]
我把arcface改成softmax，loss-type=0，
在training0.99时，eval可以到0.90了，但是如果为了得到更高的精度，还需要更精细地调参？

**Comment by wting861006** [comment]
@TccccD 我遇到了同样的问题，acc 0.97左右，verification 0.5左右，使用的arcface，换成softmax后，acc 0，lossvalue 10左右。请问你解决这个问题了吗？

**Comment by TccccD** [comment]
> @TccccD 我遇到了同样的问题，acc 0.97左右，verification 0.5左右，使用的arcface，换成softmax后，acc 0，lossvalue 10左右。请问你解决这个问题了吗？

我用的softmax，eval就上去了，arcface需要调整margin和s的大小吧，softmax精度上不去我觉得你可以试试调整下学习率等的参数看看，或者看看模型结构有没有错误
